app/src/androidTest/java/com/example/avjindersinghsekhon/minimaltodo/testApp/tests/StartTheApp.java

```diff
@@ -0,0 +1,94 @@
+package com.example.avjindersinghsekhon.minimaltodo.testApp.tests;
+
+import android.support.test.espresso.Espresso;
+
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+import static junit.framework.Assert.assertTrue;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class StartTheApp extends EspressoTestBase {
+    @Test
+    public void testAhomePageValidation() {
+        // Verify: “Minimal” and “You don’t have any todos” texts are visible
+        assertTrue("Minimal is not displayed", Home.isMinimalVisible());
+        assertTrue("You don’t have any todos is not displayed", Home.isNoToDosText());
+
+        // Verify: “Minimal” text is displayed in a toolbar
+        assertTrue("Minimal is not displayed", Home.isMinimalInToolbar());
+
+        // Verify: An image is displayed above the “You don’t have any todos” text
+        assertTrue("Image is not displayed above", Home.isImageAbove());
+
+        // Verify: "Add" button is displayed and clickable
+        assertTrue("Add button is not displayd", Home.isAddButton());
+        assertTrue("Add button is not clickable", Home.isAddButtonClickable());
+
+        // Verify: “More options” button is displayed and it has a sibling with the text "Minimal"
+        assertTrue("More option is not displayd", Home.isMoreOptions());
+        assertTrue("More option doesen't have a sibling", Home.isMoreOptionSibling());
+    }
+
+    @Test
+    public void testBbuttonsValidation() {
+        // Verify: “Minimal” text is displayed
+        assertTrue("Minimal is not displayed", Home.isMinimalVisible());
+
+        // Step: Click "Add" button
+        Home.clickAddButton();
+
+        // Veryfy: "X" button is visible
+        assertTrue("X button is not displayed", AddItem.isXButton());
+
+        // Click "X" button
+        AddItem.clickXButton();
+
+        // Verify: “You don’t have any todos” text is visible
+        assertTrue("You don’t have any todos is not displayed", Home.isNoToDosText());
+    }
+
+    @Test
+    public void testCaddItem() {
+        // Verify: "Minimal" text is visible
+        assertTrue("Minimal is not displayed", Home.isMinimalVisible());
+
+        // Step: Click "Add" button and check if FloatingActionButton is visible
```

Comment:
"// Verify: FloatingActionButton is visible"

Reply:
Done.